dependencies.gradle

```diff
@@ -5,4 +5,6 @@ dependencies {
     implementation('com.github.GTNewHorizons:GTNHLib:0.6.14:dev')
 
     shadowImplementation('org.apache.commons:commons-csv:1.10.0')
+
+    implementation 'maven.modrinth:forge-essentials:7.5.1'
```

Comment:
shouldn't this be compileOnly?  I believe it'll get pulled in while running in dev for other mods that depend on nei.

Reply:
It's also causing issues with the tests due to mod load order w/ unimixins.  Should be fine with it as compileOnly

Reply:
yea. it should